File: token-validation-core/src/main/java/no/nav/security/token/support/core/utils/EnvUtil.java
package no.nav.security.token.support.core.utils;

public
class EnvUtil {

    private
    EnvUtil() {

    }

    static final String FSS = "fss";
    static final String SBS = "sbs";
    static final String LOCAL = "local";
    static final String GCP = "gcp";
    static final String TEST = "test";
    static final String DEV = "dev";
    static final String PROD = "prod";
    static final String DEV_GCP = join(DEV, GCP);
    static final String PROD_GCP = join(PROD, GCP);
    static final String PROD_SBS = join(PROD, SBS);
    static final String DEV_SBS = join(DEV, SBS);
    static final String PROD_FSS = join(PROD, FSS);
    static final String DEV_FSS = join(DEV, FSS);
    static final String NAIS_CLUSTER_NAME = "NAIS_CLUSTER_NAME";

    private static String join(String env, String cluster) {
        return env + "-" + cluster;
    }

}

File: token-validation-core/src/main/kotlin/no/nav/security/token/support/core/utils/EnvUtil.kt
package no.nav.security.token.support.core.utils

object EnvUtil {

    const val FSS = "fss"
    const val SBS = "sbs"
    const val LOCAL = "local"
    const val GCP = "gcp"
    const val TEST = "test"
    const val DEV = "dev"
    const val PROD = "prod"

    @JvmField
    val DEV_GCP = "$DEV-$GCP"

    @JvmField
    val PROD_GCP = "$PROD-$GCP"

    @JvmField
    val PROD_SBS = "$PROD-$SBS"

    @JvmField
    val DEV_SBS = "$DEV-$SBS"

    @JvmField
    val PROD_FSS = "$PROD-$FSS"

    @JvmField
    val DEV_FSS = "$DEV-$FSS"
    const val NAIS_CLUSTER_NAME = "NAIS_CLUSTER_NAME"
}